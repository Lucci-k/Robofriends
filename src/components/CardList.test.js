import React from 'react'
import {shallow} from 'enzyme';
import CardList from './CardList';
import renderer from 'react-test-renderer';

// it('expect to render Card component', () => {
//     expect(shallow(<Card />)).toMatchSnapshot();
// })

it('CardList renders correctly', () => {
  const mockRobots = [
    {
      id: 1,
      name: 'John Snow',
      username: 'SnowyJohns',
      email: 'snow@gmail.com'
    }
  ]
  const tree = renderer
    .create(shallow(<CardList robots={mockRobots}/>))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
