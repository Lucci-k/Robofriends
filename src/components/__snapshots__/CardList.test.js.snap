// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList renders correctly 1`] = `
<div
  className="bg-light-green tc dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/1?size=200x200"
  />
  <div>
    <h2>
      John Snow
    </h2>
    <p>
      snow@gmail.com
    </p>
  </div>
</div>
`;
